helm/prometheus-rules/templates/recording-rules/cortex.rules.yml

```diff
@@ -82,8 +82,8 @@ spec:
       record: aggregation:kubelet:version
     - expr: sum(apiserver_request_total) by (cluster_type, cluster_id)
       record: aggregation:kubernetes:apiserver_request_count
-    - expr: sum(apiserver_request_latencies_bucket) by (le, cluster_type, cluster_id)
-      record: aggregation:kubernetes:apiserver_request_latencies_bucket
+    - expr: sum(apiserver_request_duration_seconds_bucket) by (le, cluster_type, cluster_id)
```

Comment:
@giantswarm/team-atlas any way how we can check that grafana.com dashboards are using the old metric name?

Reply:
Not really, some dashboards are in https://github.com/giantswarm/dashboards/. So you could grep in there, but we have more dashboards on grafana.com

Reply:
Thanks. I couldn't find it there.